pest/src/span.rs

```diff
@@ -61,6 +61,40 @@ impl<'i> Span<'i> {
         }
     }
 
+    /// Attempts to create a new span based on a sub-range.
+    ///
+    /// ```
+    /// use pest::Span;
+    /// let input = "Hello World!";
+    /// let world = Span::new(input, 6, input.len()).unwrap();
+    /// let orl = world.get(1..=3);
+    /// assert!(orl.is_some());
+    /// assert_eq!(orl.unwrap().as_str(), "orl");
+    /// ```
+    ///
+    /// # Examples
+    pub fn get(&self, range: impl std::ops::RangeBounds<usize>) -> Option<Span<'i>> {
```

Comment:
this may need to be feature-guarded in order not to break no_std support:
```suggestion
    #[cfg(feature = "std")]
    pub fn get(&self, range: impl std::ops::RangeBounds<usize>) -> Option<Span<'i>> {
```

Reply:
Oops! I had checked `core::ops::RangeBounds` existed but I seem to have forgotten to make the change. I switched them